### PR TITLE
Add vlan test to t0-2vlans test job

### PR DESCRIPTION
**Pull request by xwjiang-ms**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311

#### What is the motivation for this PR?
Elastictest performs well in distribute running PR test in multiple KVMs, which support us to add more test scripts to PR checker
#### How did you do it?
Add vlan test to t0-2vlans test job

### CODE REVIEW

**Comment by yutongzhang-microsoft** [comment]
LGTM

**Review by yutongzhang-microsoft — APPROVED** [review]

File: .azure-pipelines/pr_test_scripts.yaml [hunk 1]
```diff
@@ -274,6 +274,9 @@ onboarding_t0:
   - platform_tests/test_sequential_restart.py
   - platform_tests/test_xcvr_info_in_db.py
 
+onboarding_t0-2vlans:
+  - vlan/test_host_vlan.py
+  - vlan/test_vlan_ping.py
```
Comment (wangxin) [inline .azure-pipelines/pr_test_scripts.yaml:279]: We do not have many scripts to be covered in t0-2vlans scenario. Probably not necessary to add a new optional checker for t0-2vlans. Can you just add these 2 scripts to the existing t0-2vlans checker? The new optional job is not required IMO.

Comment (xwjiang-ms) [reply .azure-pipelines/pr_test_scripts.yaml:279]: For these 2 tests, it maybe fine, but we still have a pending script test_vlan.py, it's continuously failing on t0-2vlans, so once we fixed error and add it to PR test, we need to observe some time before adding to t0-2vlans checker.
So I think if adding a new checker is needed in the future, why not add it now?

Comment (xwjiang-ms) [reply .azure-pipelines/pr_test_scripts.yaml]: Fixed

**Review by wangxin — APPROVED** [review]

**Pull request merged**